src/CppUTestExt/MockSupport_c.cpp

```diff
@@ -72,28 +72,36 @@ static MockExpectedCall* expectedCall = NULL;
 static MockActualCall* actualCall = NULL;
 static MockFailureReporterForInCOnlyCode failureReporterForC;
 
-class MockCFunctionComparatorNode : public MockNamedValueComparator
+class MockCFunctionComparatorAndCopierNode : public MockNamedValueComparator, public MockNamedValueCopier
```

Comment:
Any way we can avoid the use of multiple inheritance here? I think this is the only place in CppUTest and prefer not using it.